sign_oca/controllers/main.py

```diff
@@ -104,3 +104,60 @@ def get_sign_oca_sign_access(
         return signer_sudo.action_sign(
             items, access_token=access_token, latitude=latitude, longitude=longitude
         )
+
+    def get_sign_requests_domain(self, request):
+        domain = [
+            ("request_id.state", "=", "sent"),
+            ("partner_id", "child_of", [request.env.user.partner_id.id]),
+            ("signed_on", "=", False),
```

Comment:
```suggestion
            ("request_id.state", "in", ("sent", "signed")),
            ("partner_id", "child_of", [request.env.user.partner_id.id]),
```
IMO this should be the appropriate domain for displaying all pending and signed requests.

The texts on the portal should probably be more consistent to take this into account.